src/cf-layout/src/cf-layout.less

```diff
@@ -545,29 +545,29 @@
     border-left-width: @grid_gutter-width / 2;
 
     &:before {
-        content: "";
         display: block;
         height: 1px;
-        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
-        background-color: @grid_column__top-divider;
-        position: static;
         width: 100%;
         margin-left: auto !important;
+        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
Bottom should come before left, following the order of shorthand dimension declarations (top, right, bottom, left).